src/applications/disability-benefits/686c-674/config/chapters/household-income/helpers.js

```diff
@@ -20,7 +20,7 @@ export const netWorthCalculation = (
 
 export const netWorthTitle = (
   <p>
-    Did the household have a <strong>net worth</strong> greater than $129,094 in
+    Did the household have a <strong>net worth</strong> greater than $130,733 in
```

Comment:
Would this work better if it was moved into the `constants` file (if you have one)?